libraries/chain/include/eosio/chain/block_header_state.hpp

```diff
@@ -187,24 +188,24 @@ FC_REFLECT_DERIVED(  eosio::chain::block_header_state, (eosio::chain::detail::bl
 
 
 FC_REFLECT( eosio::chain::legacy::snapshot_block_header_state_v2::schedule_info,
-          ( schedule_lib_num )
-          ( schedule_hash )
-          ( schedule )
+          (schedule_lib_num)
```

Comment:
Were you just cleaning this up, or is this to avoid problems in the regex?  If it is to avoid problems in the regex, I'd like to look at them so that people can use whatever whitespace they want.

Reply:
updated script to handle this rather than change code format